<?xml version="1.0"?>
<launch>
    <!--Launch the move-base node to create global and local planner-->
    <node pkg="move_base" type="move_base" respawn="false" name="move_base">
    <rosparam file="$(find f1tenth_gym_ros)/quali/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find f1tenth_gym_ros)/quali/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find f1tenth_gym_ros)/quali/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find f1tenth_gym_ros)/quali/config/global_costmap_params.yaml" command="load" />
    <!--rosparam file="$(find f1tenth_gym_ros)/quali/config/teb_local_planner_params.yaml" command="load" /-->

    <param name="base_global_planner" value="global_planner/GlobalPlanner" />
    <param name="planner_frequency" value="20.0" />
    <param name="planner_patience" value="10.0" />
	<!--param name="cost_factor" value="0.6" /-->
	<param name="neutral_cost" value="75" />
    <!--param name="use_dijkstra" value="false" /-->

    <param name="clearing_rotation_allowed" value="false" />
    </node>
	
	<!--Create path generation node to find available goal point-->
	<node pkg="f1tenth_gym_ros" type="path_generation_fast.py" name="path_generation"/>

	<!--Create pure pursuit node to track path-->
	<node pkg="f1tenth_gym_ros" type="path_pursuit_fast.py" name="pursuit_path" output="screen"/>
	
	<include file="$(find f1tenth_gym_ros)/launch/gym_bridge_host.launch"/>

</launch>
